_posts/2015-07-20-self-organizing-team-2-footballs.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+author: curiousagilist
+authorname: Bob Allen
+title: Self Organizing Team Part 2 - Footballs
+preview: 'Below is a handout that was given at the beginning of the day. A second handout was given just after lunch. Together they define an exercise designed to require teams to work out and demonstrate these skills'
+---
+Below is a handout that was given at the beginning of the day. A second handout was given just after lunch. Together they define an exercise designed to require teams to work out and demonstrate these skills:
+
+
+1. Self organization of teams when given full reign (and full responsibility).
+2. How to work with what you're given, even when extremely sparse.
+3. How to deliver _iteratively (MVP)_.
+4. How to keep parts of the solution decoupled from one another.
+
+&lt;Handout #1 - Morning session&gt;
```

Comment:
That library I linked is a set of React components, which is not really suited for the current state of coderetreat.org. There are probably CSS libraries out there that implement a similar effect. Otherwise, we could whip a quick CSS class of our own to do it.